libportapat/porta_paths/paths.go

```diff
@@ -0,0 +1,86 @@
+package porta_paths
```

Comment:
Setting custom paths can be done with standard XDG environment variables (`XDG_DATA_HOME`, `XDG_CONFIG_HOME` and `XDG_STATE_HOME`).

This should eliminate the need for the `porta_paths` package I believe?